data/human/human missions.txt

```diff
@@ -5547,3 +5547,130 @@ mission "Quicksilver Mixup 1"
 			
 
 event "quicksilver mixup 2 available"
+
+
+
+mission "Argosy Hijacking (Trigger)"
+	invisible
+	source
+		government "Syndicate"
+	to offer
+		random < 10
+	on offer
+		event "argosy hijacking trigger" 14 35
+		fail
+
+event "argosy hijacking trigger"
+
+mission "Argosy Hijacking"
+	name "Stop ship hijacking"
+	description "The <npc> was hijacked on <origin>; disable it and return to <origin>. Do not capture or destroy it; Syndicated Security will deal with the hijackers in space after you have disabled it."
+	landing
+	source
+		government "Syndicate"
+		not attributes "station"
+		near "Alcyone" 4
+		near "Almach" 3
+		near "Gienah" 3
+	to offer
+		has "Argosy Hijacking (Trigger): offered"
+		has "event: argosy hijacking trigger"
+	on offer
+		conversation
+			`You're signing the landing papers with the Syndicated Security officer on <origin>, a standard procedure after landing in a planet within the Syndicate bloc. "Everything seems to be in order. And of course, we'll have your ship refuelled promptly. Have a good day, Captain <last>." He nods his head at you and starts to move off towards the next ship.`
+			`	Just before he is able to, however, you hear a commotion going on by the <npc model> on the landing pad next to you.`
+			choice
+				`	(Turn around to see what's going on.)`
+					goto turn
+				`	(Ignore the commotion.)`
+					goto ignore
+			label turn
+			`	Both of you turn your head to see what's happening, but as you do, there's a deafening blare of warning sirens through <origin>'s spaceport. Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`
+				goto hijack
+			
+			label ignore
+			`	Figuring it's nothing, you turn back towards the <ship> only to hear a deafening blare of warning sirens through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`
```

Comment:
"Sirens" seems unnecessary to me.
```suggestion
			`	You turn back towards the <ship> only to hear a deafening blare of warning through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`
```

Reply:
How about:
```suggestion
			`	You turn back towards the <ship> only to hear a deafening blare of warning sounds through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`
```

Reply:
Are you using sounds as a verb or noun here?

Reply:
Noun.

Reply:
Okay, I see how you might have misread this. I'm changing it back to sirens, though.